### DISCUSSION TITLE
Is a partial Wet Bulb forecast (for 7 hours) expected in some cases or a bug?

### DISCUSSION THREAD

**Discussion by t1m0thy** [weather-gov/api#672] [Q&A]
Hello, I noticed for a grid point that wet bulb data is only reported for about 7 hours.  Normally I see it alongside all other data.  I am wondering if this should be expected in some cases or worth bringing to someone's attention?  The example is from `https://api.weather.gov/gridpoints/BTV/108,19`

where this is returned for wet bulb
```
"wetBulbGlobeTemperature": {
            "uom": "wmoUnit:degC",
            "values": [
                {
                    "validTime": "2023-11-19T08:00:00+00:00/PT1H",
                    "value": -2.7777777777777777
                },
                {
                    "validTime": "2023-11-19T09:00:00+00:00/PT3H",
                    "value": -2.2222222222222223
                },
                {
                    "validTime": "2023-11-19T12:00:00+00:00/PT1H",
                    "value": -1.6666666666666667
                },
                {
                    "validTime": "2023-11-19T13:00:00+00:00/PT1H",
                    "value": -3.3333333333333335
                },
                {
                    "validTime": "2023-11-19T14:00:00+00:00/PT1H",
                    "value": -2.7777777777777777
                }
            ]
        },
```

**Comment by dwhitemv25** [comment]
For my gridpoint, I'm getting 179 hours of wet bulb forecast, so the API can provide larger time ranges. I suspect your WFO must not be sending that data in their forecast packages. I'd contact the WFO and ask them the question.

**Comment by t1m0thy** [comment]
I’ve seen 179 hours on other locations, too.  That’s good advice to contact WFO, thanks! On Nov 21, 2023, at 18:14, dwhitemv25 ***@***.***> wrote:﻿
For my gridpoint, I'm getting 179 hours of wet bulb forecast, so the API can provide larger time ranges. I suspect your WFO must not be sending that data in their forecast packages. I'd contact the WFO and ask them the question.
